spec/integration/users_index_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+
+RSpec.describe 'users#index', type: :feature do
+  describe 'User' do
+    before(:each) do
+      @mohammed = User.create!(
+        name: 'mohammed',
+        photo: 'msMod.jpeg',
+        bio: 'Lorem ipsum dolor sit amet',
+        posts_counter: 0
+      )
+
+      @tom = User.create!(
+        name: 'tom',
+        photo: 'msMod.jpeg',
+        bio: 'Lorem ipsum dolor sit amet',
+        posts_counter: 0
+      )
+
+      visit root_path
+    end
+
+    it 'Shows the username' do
+      expect(page).to have_content('mohammed')
+    end
+
+    it "Shows the user's photo" do
+      all('img').each do |i|
+        expect(i[:src]).to eq('msMod.png')
+      end
+    end
```

Comment:
- [ ] At the moment, this test is not passing because the photo format does not match any of the profiles created. Please ensure that this test passes.